android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -194,10 +211,70 @@ public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request
         }
       }
     });
+  }
 
+  private void loadCustomWebView() {
+    webView.setWebChromeClient(new BridgeWebChromeClient(this));
 
-    // Get to work
-    webView.loadUrl(appUrl);
+    // Saving off old client so no functionality is lost if
+    // custom webview implements an existing client
+    final WebViewClient oldClient = webView.getWebViewClient();
+    webView.setWebViewClient(new WebViewClient() {
+
+      @Override
+      public void onPageStarted(WebView view, String url, Bitmap favicon) {
+        if (oldClient != null) {
```

Comment:
What's the purpose of this override?

Reply:
TurboLinks Android has its own `WebClient` that it depends on so overriding it with completely makes it 💣 This allows both clients to be called. The `oldClient` which is the one TurboLinks created and then this one which injects the Capacitor javascript and whatnot

Reply:
`getWebViewClient()` requires API 26, but capacitor supports 21, so this won't work in a great part of devices.

Do you plan to continue with this or I can close the PR?